app/scripts/models/account.js

```diff
@@ -46,6 +46,7 @@ define([
 
   var DEFAULTS = _.extend({
     accessToken: undefined,
+    declinedSyncEngines: undefined,
```

Comment:
Are these values stored, or will changing this data require the user visit `about:preferences#sync`?